scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/PipelineSubscriber.kt

```diff
@@ -136,6 +137,7 @@ class PipelineSubscriber(
                 updateEventReason = "kafka topic error"
             }
         } else {
+            pipelineStarted = true
             logger.warn("pipeline ${metadata.id} already exists")
```

Comment:
Should we put log the pipeline name here instead of id? or perhaps the pair (name, id). Just to make it more readable